config/initializers/i18n.rb

```diff
@@ -0,0 +1,8 @@
+# Where the I18n library should search for translation files
+I18n.load_path += Dir[Rails.root.join('config', 'locales', '**', '*.{rb,yml}')]
+
+# Permitted locales available for the application
+I18n.available_locales = %i[en pt-BR es]
+
+# Set default locale to something other than :en
+I18n.default_locale = ENV['LANGUAGE']
```

Comment:
Esse aqui é legal que já coloca um valor default caso não tenha nada no env
```suggestion
I18n.default_locale = ENV.fetch('LANGUAGE', 'en')
```

Reply:
Boa!